cucumber-expressions/javascript/scripts/postinstall.js

```diff
@@ -0,0 +1,10 @@
+var path = require('path')
+var fs = require('fs')
+var exec = require('child_process').exec
+
+fs.exists(path.join(__dirname, '..', 'dist'), function (err) {
+  if (err) throw err;
+  exec('npm run build', {cwd: path.join(__dirname, '..')}, function (err) {
+    if (err) throw err;
+  })
+})
```

Comment:
Note the guard checking for the presence of `dist`. The `npn run build` command will not run when the package is published (it would have been done in `prepublish`)

If you know a better technique that allows code to depend on unreleases ES6 modules I'd love to know.